civiclens/nlp/pipeline.py

```diff
@@ -0,0 +1,89 @@
+import datetime
+
+from ..utils.database_access import Database, pull_data
+from . import comments, titles
+
+
+def doc_generator(df):
+    for row in df.iter_rows():
+        yield (row)
+
+
+if __name__ == "__main__":
+    # get time nlp last updated
+    nlp_updated_query = """SELECT nlp_last_updated
+                        FROM regulations_nlpoutput
+                        ORDER BY nlp_last_updated ASC LIMIT 1"""
+
+    db_date = Database()
```

Comment:
My logic here was that I didn't want to leave a connection open very long because of the issues that Michael said can happen with multiple connections and people editing the database. Can definitely change this but lets backlog for now